src/renderer/components/deposit/withdraw/Withdraw.tsx

```diff
@@ -1,65 +1,282 @@
+        return <Styled.FeeErrorLabel key="asset-fee-out-error">{msg}</Styled.FeeErrorLabel>
+      }),
+      O.getOrElse(() => <></>)
+    )
+  }, [oAssetChainFee, intl, asset, withdrawAmounts])
+
+  const isThorOutFeeError = useMemo(() => {
+    if (withdrawPercent <= 0) return false
+    return FP.pipe(
+      oFees,
+      O.fold(
+        // Missing (or loading) fees does not mean we can't sent something. No error then.
+        () => !O.isNone(oFees),
+        ({ thorOut }) => assetToBase(withdrawAmounts.runeWithdraw).amount().isLessThan(thorOut.amount())
+      )
+    )
+  }, [oFees, withdrawAmounts.runeWithdraw, withdrawPercent])
+
+  const renderThorOutFeeError = useMemo(() => {
+    return FP.pipe(
+      oFees,
+      O.map(({ thorOut }) => {
+        const msg = intl.formatMessage(
+          { id: 'deposit.withdraw.add.error.outFeeNotCovered' },
+          {
+            fee: formatAssetAmountCurrency({
+              amount: baseToAsset(thorOut),
+              asset: AssetRuneNative,
+              trimZeros: true
+            }),
+            amount: formatAssetAmountCurrency({
+              amount: withdrawAmounts.runeWithdraw,
+              asset: AssetRuneNative,
+              trimZeros: true
+            })
+          }
+        )
+        // `key`  has to be set for any reason to avoid "Missing "key" prop for element in iterator"
+        return <Styled.FeeErrorLabel key="thor-fee-out-error">{msg}</Styled.FeeErrorLabel>
+      }),
+      O.getOrElse(() => <></>)
+    )
+  }, [oFees, intl, withdrawAmounts.runeWithdraw])
+
+  const renderFee = useMemo(() => {
+    const loading = <>...</>
+    return FP.pipe(
+      fees,
+      RD.fold(
+        () => loading,
+        () => loading,
+        (error) => (
+          <>
+            {intl.formatMessage({ id: 'common.error' })} {error?.message ?? ''}
+          </>
+        ),
+        ({ thorMemo, thorOut, assetOut }) => {
+          const formattedThorMemoFee = formatFee({
+            amount: thorMemo,
+            asset: AssetRuneNative
+          })
+          const formattedAssetOutFee = formatFee({
+            amount: calculateAssetChainFee(assetOut),
+            asset
+          })
+          const formattedThorOutFee = formatFee({
+            amount: thorOut,
+            asset: AssetRuneNative
+          })
+
+          if (!isDesktopView) {
+            return `${intl.formatMessage({
+              id: 'common.fees'
+            })}: ${formattedThorMemoFee} + ${formattedThorOutFee} + ${formattedAssetOutFee}`
+          }
+
+          return (
+            <>
+              {intl.formatMessage(
+                { id: 'deposit.withdraw.fees' },
+                { thorMemo: formattedThorMemoFee, assetOut: formattedAssetOutFee, thorOut: formattedThorOutFee }
+              )}
+            </>
+          )
+        }
+      )
+    )
+  }, [asset, calculateAssetChainFee, fees, intl, isDesktopView])
```

Comment:
as we want to keep things simplier why don't we use `Fees` component ? detailed information we can place behind `i` popup icon. we can put this "additional info"  as a part of `Fees` component to have the same things everywhere

Reply:
We can go with `Fees` component later if all implementations of fees will be done and we know what we really need in this component.

Reply:
that's totally correct. maybe it's better to keep things simplier until we 100% understand what we want form the UI ? 
Fees component allows to gather all logic in a single place with a single logic. in case we need custom fees rendering we can implement it later. what do you think ?

Reply:
We already have a custom case here by providing different labels for desktop / mobile view. We don't have that in `Fees` (yet) and we should wait if it make sense for other parts, too. Let's just wait few days, then we start another round of `Fees` component (if needed).

Reply:
Fee labels for WITHDRAW:

**Desktop view**
![Screenshot from 2020-12-14 19-48-49](https://user-images.githubusercontent.com/61792675/102122335-9e093080-3e45-11eb-8d94-fb7569b8369c.png)

**Mobile view**

![Screenshot from 2020-12-14 19-49-17](https://user-images.githubusercontent.com/61792675/102122350-a3667b00-3e45-11eb-8466-1fc189160bb3.png)

Reply:
and we still can have "mobile view" for all devices and (i) button with additional description about fees if semeone wants to see detailed info